smriprep/cli/run.py

```diff
@@ -131,6 +117,9 @@ def get_parser():
     g_other = parser.add_argument_group('Other options')
     g_other.add_argument('-w', '--work-dir', action='store', type=Path, default=Path('work'),
                          help='path where intermediate results should be stored')
+    g_other.add_argument(
+        '--force-build', action='store_true', default=False,
```

Comment:
i'm not a huge fan of the name, wdyt about `--overwrite` or `--force-rerun`
also needs a help string, pulled from below:
>Do not fast-track the workflow by searching for existing derivatives.

Reply:
I'm going for a white-list approach switching over to ``--fast-track``